letters/original/16-letters.adoc

```diff
@@ -451,16 +473,22 @@ manifested in and through his beloved Son Jesus Christ, to whom be ascribed glor
 salvation and honour.
 Amen.
 
+[.signed-section-closing]
 Your friend and sister, I hope, in the saints`' travail and tribulation,
```

Comment:
like in the modernized, this one bumped onto a second line. I think we should make it part of the paragraph above.